dkg/frost/timer.go

```diff
@@ -0,0 +1,91 @@
+package frost
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+)
+
+type RoundTimeoutFunc func(ProtocolRound) time.Duration
+type OnTimeoutFn func() error
+
+var defaultTimeout int = 2
+var defaultTimeDuration = time.Second
```

Comment:
2 seconds is too short. A more reasonable timeout is `10 mins`, `30 mins` or `1 hour`. The idea is if anything is wrong with the protocol instance, it will be handled eventually so the peers won't wait indefinitely.

Reply:
I've changed the default timeout to 10 min, do you think we should take a timeout as a parameter when we initialize node or runner?